client/src/pages/Student.js

```diff
@@ -141,6 +155,7 @@ class Student extends Component {
                       value={this.state.student.hoursAvail}
                       onChange={this.changeHandler}
                   />
+                  {this.validator.message('hoursAvail', this.state.student.hoursAvail, 'required|integer|min:10|max:20')}
```

Comment:
min and max are still constraints on the length of the input, but not the value of the input

Reply:
@zhao926. "required|integer|min:10|max:20" ensures that the value entered is a number or integer between 10 and 20. Tobi's current implementation is correct. see [Library](https://www.npmjs.com/package/simple-react-validator#integer)  documentation for reference